buildenv/docker/mkdocker.sh

```diff
@@ -569,7 +569,7 @@ bootjdk_url() {
 }
 
 install_bootjdks() {
-  local versions="8 11 13 14"
+  local versions="8 11 14"
```

Comment:
`defaults.yml` still says:
```
  boot_jdk:
    14: '/usr/lib/jvm/adoptojdk-java-13'
```
So I don't think it's yet time to remove jdk13 here.

Reply:
I'll open a PR to change it. Just because OpenJ9 builds still use 13 doesn't mean we can't tell users to use 14.

Reply:
As long as OpenJ9 builds (that could use the docker image) use 13, it should stay.
I wasn't objecting to the other file that advises using 14.

Reply:
I don't think there is any relation between defaults.yml and running in docker atm. 
Created https://github.com/eclipse/openj9/pull/9422